internal/services/contracts_dcn_handler.go

```diff
@@ -55,3 +55,27 @@ func (c *ContractsEventsConsumer) handleNewDCNExpiration(ctx context.Context, e
 
 	return nil
 }
+
+func (c *ContractsEventsConsumer) handleNameChanged(ctx context.Context, e *ContractEventData) error {
+	eventName := NameChanged.String()
+	logger := c.log.With().Str("EventName", eventName).Logger()
+
+	var args DCNNameChangedEventData
+	if err := json.Unmarshal(e.Arguments, &args); err != nil {
+		return err
+	}
+
+	dcn := models.DCN{
+		Node: args.Node,
+		Name: null.StringFrom(args.Name),
+	}
+
+	_, err := dcn.Update(ctx, c.dbs.DBS().Writer, boil.Whitelist(models.DCNColumns.Name))
+	if err != nil {
+		return err
+	}
+
+	logger.Info().Str("Node", string(args.Node)).Msg(eventName + " Event processed successfuly")
```

Comment:
`string(node)` is usually not going to be meaningful. Do you want to try `.Bytes("node", args.Node)`?

Reply:
Alternatively just something like [`hexutil.Encode`](https://pkg.go.dev/github.com/ethereum/go-ethereum/common/hexutil#Encode).

Reply:
Like it produces stuff like

```
{"level":"info","EventName":"NewNode","Node":"fe%]\ufffd,\ufffdN\ufffd\ufffd\u0006B\ufffd\ufffd\ufffd\f\ufffdJ\ufffd\u001d\ufffd\ufffdl\ufffd\ufffd8\u000fg\ufffd\ufffdW\ufffd","message":"NewNode Event processed successfuly"}
```